src/site/content/en/blog/isinputpending/index.md

```diff
@@ -0,0 +1,194 @@
+---
+title: Better JS scheduling with isInputPending()
+subhead: >
+  A new JavaScript API that may help you avoid the tradeoff between
+  load performance and input responsiveness.
+description: >
+  A new JavaScript API that may help you avoid the tradeoff between
+  load performance and input responsiveness.
+date: 2020-11-13
+hero: hero.jpg
+alt: A traffic sign that says 'GIVE WAY'.
+tags:
+  - blog
+  - javascript
+  - performance
+  - rendering
+authors:
+  - nateschloss
+  - andrewcomminos
+---
+
+Loading fast is hard. Sites that leverage JS to render their content currently
+have to make a trade-off- either perform all the work needed for display all at
+once, or chunk the work into smaller tasks in order to remain responsive to
+input and paint -- occasionally, at the expense of performance.
+
+To eliminate the need to make this trade-off, Facebook proposed and implemented
+the `isInputPending()` API in Chrome in order to improve responsiveness without
```

Comment:
```suggestion
the `isInputPending()` API in Chromium in order to improve responsiveness without
```

Unless this is actually only for Chrome and not for other Chromium-based browsers, we should use Chromium when referring to the actual browser (we can use Chrome to refer to the team that you collaborated with)